index.html

```diff
@@ -10,7 +10,7 @@
     <div id='head'>
       <div id='intro' class='center'>Enter total workout time in minutes (rests included):</div>
       <div class='center'>
-        <input id='minutes' value='20' type='number'/>
+        <input id='minutes' value='20' type='range' min='1' max='100'/>
```

Comment:
hi caleb! i like this idea but would like to keep the ability to use fractional values like 0.5 for testing. so maybe min='0' or would solve it.

Reply:
it'd also be nice if it was possible to actually enter/see the time in the input... probably what we want here is a text input field but with some validation